src/Events/Migration/FunctionsAndProcedures/createlogspartition.sql

```diff
@@ -0,0 +1,44 @@
+CREATE OR REPLACE PROCEDURE events.deletesubscription(_id integer)
```

Comment:
Something wrong here. The new function is defined inside a script for an existing function.